docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -6,15 +6,62 @@ checked: true
 ---
 # Command Line Interface
 
+This topic provides information on how to run TestCafe tests from a command line.
+
+## Installation
+
+You can install TestCafe globally, or locally to your project.
+
+### Globally
+
+```bash
+npm install -g testcafe
+```
+
+After that, you can run TestCafe by using the `testcafe` command (see [Usage](#usage)).
+
+```bash
+testcafe safari tests/
+```
+
+### Locally
+
+The following command will install TestCafe into your project directory and save it in the dependencies list.
+
+```bash
+npm install --save-dev testcafe
+```
+
+The local installation has two advantages:
+
+* It makes your project setup easier: running `npm install` in the project directory will automatically install TestCafe as well.
+* Different projects can depend on different versions of TestCafe.
+
+You can run the local version of TestCafe in one of the following ways.
+
+* From the `node_modules` folder.
+
+    ```bash
+    node node_modules/testcafe/bin/testcafe safari tests/
```

Comment:
This is not how it used usually. You need to add it to [npm scripts](https://docs.npmjs.com/misc/scripts)